src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,112 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String SET_UP_NAME = "Travis";
+    private static final String SET_UP_PASSWORD = "123456789";
+    private static final int SET_UP_AGE = 55;
+    private final RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+    private User user;
+    private StorageDaoImpl storageDao;
```

Comment:
let's not create storageDao instance.  I recommend using Storage class directly for your purposes